index.bs

```diff
@@ -742,6 +747,15 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 			1. Return |p|.
 
+		<h4 method for=ClipboardItem lt=supports(type)>supports(type)</h4>
+
+			This method must run the below steps:
+
+			1. If |type| is in [=mandatory data types=], then return true.
+
+			1. If |type| is not in [=mandatory data types=], then return false.
```

Comment:
Maybe this can be just "If not"?

BTW, does this mean the API supports only "mandatory data types" and not anything else?

Reply:
That is a good point. We [removed the types](https://github.com/w3c/clipboard-apis/pull/155) that may be supported by a user agent, but I think it would be good to have that list as we're adding this new API to check supported formats. I'll add it in a separate section. Thanks for reviewing this PR!